Shoko.Plugin.Abstractions/DataModels/IAniDBFile.cs

```diff
@@ -17,9 +17,9 @@ public interface IAniDBFile
         /// </summary>
         string Source { get; }
         /// <summary>
-        /// The Filename as released, according to AniDB. It's usually correct.
+        /// The file name used internally by AniDB. It has a consistant format.
```

Comment:
![image](https://github.com/ShokoAnime/ShokoServer/assets/994214/5ff66c8d-04e8-4faf-a53e-597d0e339250)
Are you sure this is even consistent?

Reply:
older versions of shoko overwrote the file name locally with the file name at the time of linking. as for newer names, i simply don't know.

Reply:
Something changed in how Shoko stored the data, it seems that Shoko wasn't using what was returned by AniDB before.   None of those files appear to be in the format AniDB returns.

Reply:
Then simply put, we can't trust that to be the case in all cases, since we have people with Shoko instances that old, we need to consider these.

Telling people to re-import all their data isn't going to be feasible, or reasonable at the least, without us having a way of correcting this to the "correct" data.